src/screens/Customers/AddCustomer.tsx

```diff
@@ -108,7 +117,7 @@ function AddCustomer(props: AddCustomerProps) {
     const customer = JSON.parse(JSON.stringify(EMPTY_CUSTOMER));
     customer.name = customerName;
     customer.customerNumber = parseInt(customerNumber);
-    customer.meterType = selectedMeterType;
+    customer.meterType = revMeterTypeMap.get(selectedMeterType); // Map back to English
     customer.meterNumber = isNaN(parseInt(meterNumber)) ? null : parseInt(meterNumber);
     customer.tariffPlanId = selectedTariffPlanId;
```

Comment:
Not technically a part of this PR, but all customers made right now are default inactive on the frontend. Could you fix that by adding a `customer.isactive = true`